tpgtools/types/constants.go

```diff
@@ -0,0 +1,12 @@
+package types
```

Comment:
nit: I'm not in love with `types` as a package name to cover overrides/resource/properties/products, I'd imagine `types` would refer to structs/functions/methods that operate on the type of a property. Maybe `model` or `resourcemodel`, since this package contains the structs used to model a resource?

(sidebar: tpgtools is effectively a model-view-controller where the model is the `Resource` type, the `view` is the `.go.tmpl` file(s) and the controller is the generator)

Reply:
agree `model` would be a better name. Hesitant to use resource model